libs/aws/odc/aws/queue.py

```diff
@@ -3,6 +3,55 @@
 from typing import Mapping, Any, Iterable, Optional
 
 
+def redrive_queue(
+    queue_name: str, to_queue_name: str = None, limit: int = None, dryrun: bool = False
```

Comment:
if it can be `None` it can not be `str` or `int`, needs to be `Optional[str]` and `Optional[int]`

Reply:
For `limit` changing default to `0` might be better than making `Optional[int]`